File: fizzbuzz_in_python/fizzbuzz.py
def fizz_buzz(var):
  if var%3 == 0 and var%5 == 0:
    return 'FizzBuzz'
  
  elif var%3 == 0:
    return 'Fizz'

  elif var%5 == 0:
    return 'Buzz'
  
  else:
    return var

File: basic_aritgeo/fizzbuzz.js
function fizzBuzz (num) {
  if(num % 3 === 0 && num % 5 === 0 ){
    return "FizzBuzz";
  }
  else if( num % 3 === 0){
    return "Fizz";
  }
  else if( num % 5 === 0){
    return "Buzz";
  }else{
    return num;
  }
}